githubnotifier.py

```diff
@@ -433,6 +433,10 @@ def update_feeds(self, feeds):
                  'message': message,
                  'icon': user['avatar_path']}
 
+            # If the user has no real name defined, fallback to username
+            if not n['title']:
+                n['title'] =  user.get('login', user['login'])
```

Comment:
Did you mean `user.get('login')`?

Passing in `user['login']` as the default seems problematic (ie if 'login' is not present you'd still try to get that key from the user dict)